Add doc note about filter indicator badge counts (#10177)
laravel-filament/filament: packages/tables/docs/04-filters/05-custom.md

If you are using a [custom filter form](#custom-filter-forms), you should use [`indicateUsing()`](#custom-active-indicators) to display an active indicator.

> Note: if you do not specify an indicator for your Filter, then the badge-count of how many filters are active will not increment correctly.

Reviewer: ```suggestion
Please note: if you do not have an indicator for your filter, then the badge-count of how many filters are active in the table will not include that filter.
```
